init.lua

```diff
@@ -274,6 +270,7 @@ require('lazy').setup({
     'folke/which-key.nvim',
     event = 'VimEnter', -- Sets the loading event to 'VimEnter'
     opts = {
+      delay = 0,
```

Comment:
Add a comment here explaining delay option and that it is independent of the timeoutlen option.

Reply:
Done.